examples/react-native/metro.config.js

```diff
@@ -69,7 +73,11 @@ const watchFolders = internalPackages
   .filter(isInternalUsedDep)
   .map(({ packagePath }) => packagePath);
 
-config.watchFolders = [WORKSPACE_NODE_MODULES_PATH, ...watchFolders];
+config.watchFolders = [
+  WORKSPACE_NODE_MODULES_PATH,
+  ...watchFolders,
+  ENVIRONMENTS_MODULE_PATH,
```

Comment:
Is there a reason that `ENVIRONMENTS_MODULE_PATH` is added after `watchFolders`?

Reply:
nope